files/install-ptfe.sh

```diff
@@ -290,6 +292,20 @@ if [ "x${role}x" == "xmainx" ]; then
             "--airgap-installer ${replicated_installer_path}"
         )
     fi
+
+    #If a custom weave CIDR is provided, set the necessary arguement
+    if [[ $(< /etc/ptfe/weave-cidr) != none ]]; then
+        ptfe_install_args+=(
+            "--weaveCIDR=$(cat /etc/ptfe/weave-cidr)"
+        )
+    fi
+
+    #If a custom Replicated service CIDR is provided, set the necessary argument
+    if [[ $(< /etc/ptfe/repl-cidr) != none ]]; then
```

Comment:
similarly with the other review comment, I think this will run no matter if the file is empty or not.


```suggestion
    if [[ $(< /etc/ptfe/repl-cidr) != "" ]]; then
```